RickAndMorty/Networking/RMEndpoint.swift

```diff
@@ -35,11 +38,16 @@ enum RMLocationEndpoint: RMEndpoint {
     static let domain = "https://rickandmortyapi.com/api/"
     
     case locationPage(Int)
+    case oneLocation(Int)
```

Comment:
Why are you adding endpoints for loading one (Location, character, episode) in this PR?

Reply:
I remove it